src/components/MainContent/MainContent.tsx

```diff
@@ -72,7 +72,7 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
   item: {
     transition: 'all 0.75s ease',
-    padding: '3em 0',
+    padding: '4.2em 0',
```

Comment:
@timmydoza I think switching all of the `<Grid container/>`s to plain `<div />`s bunched everything closer together? So I added a bit more top and bottom padding